tensorflow/python/training/optimizer.py

```diff
@@ -385,7 +385,7 @@ def _assert_valid_dtypes(self, tensors):
   def _valid_dtypes(self):
     """Valid types for loss, variables and gradients.
 
-    Defaults to `float32`. Subclasses should override to allow other types.
+    Subclasses should override to allow other types.
```

Comment:
Would it be more informative but still correct to say "types other than floats"?